eventually/utils/validators.py

```diff
@@ -0,0 +1,51 @@
+"""
+Project validators
+==================
+
+Module that provides validation functions for all kinds of project's data.
+"""
+
+import datetime
+
+
+def duration_validator(value):
+    """
+    Function that provides validation for time duration field.
+
+    :param value: the duration that represented as the amount of second.
+    :type value: int or float
+
+    :return: `True` if value if valid and `False` or `None` if it is not.
+    """
+
+    if not isinstance(value, int) and not isinstance(value, float):
+        return False
+
+    if value < 0:
+        return False
+
+    try:
+        datetime.timedelta(seconds=value)
+        return True
+    except OverflowError:
+        pass
+
+
+def timestamp_validator(value):
+    """
+    Function that provides validation for time date and datetime fields.
+
+    :param value: the timestamp that represented as the amount of second.
+    :type value: int or float
+
+    :return: `True` if value if valid and `False` or `None` if it is not.
+    """
+
+    if not isinstance(value, int) and not isinstance(value, float):
+        return False
+
+    try:
+        datetime.datetime.fromtimestamp(value)
+        return True
+    except (OverflowError, ValueError, OSError):
+        pass
```

Comment:
Will it change behavior in case you remove : 
`if not isinstance(value, int) and not isinstance(value, float):
         return False
`
?

Reply:
Removing of those lines make be possible TypeError trace. I have removed that lines and add cathing of TypeError exception.
Thanks)